cmd/keepalived-exporter/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"flag"
+	"net/http"
+	"os"
+
+	"github.com/cafebazaar/keepalived-exporter/internal/collector"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	listenAddr := flag.String("web.listen-address", ":2112", "Address to listen on for web interface and telemetry.")
+	metricsPath := flag.String("web.telemetry-path", "/metrics", "A path under which to expose metrics.")
+	keepalivedJSON := flag.Bool("ka.json", false, "Send SIGJSON and decode JSON file instead of parsing text files.")
+
+	flag.Parse()
+
+	coll := collector.NewKCollector(*keepalivedJSON)
+	prometheus.MustRegister(coll)
+
+	logrus.Info("Listening on address: ", *listenAddr)
```

Comment:
I don't wanna code to be mess by adding `ListenAndServe` to another go routine that I could log for that It's just an info for what's going on and if errors happened it will log for that, too.
We can change the description if it's not good enough.